ledger/src/shred/merkle.rs

```diff
@@ -994,17 +995,13 @@ fn make_shreds_data<'a>(
 }
 // Generates coding shreds for the current erasure batch.
 // Updates ShredCommonHeader.index for coding shreds of the next batch.
-fn make_shreds_code(
+fn make_code_shred_blanks(
```

Comment:
nit: maybe `make_shreds_code_empty_payload` or `make_shreds_code_header_only`?

`make_shreds_` prefix is a little more consistent with other functions.

`code_empty_payload` is a little more descriptive given it is important we populate the header portion of the shreds.

Reply:
Done in https://github.com/anza-xyz/agave/pull/6077/commits/d61d05153e1711b2385bab947f853fdcfc116397